cinema/serializers.py

```diff
@@ -33,7 +33,16 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
+        )
+        read_only_fields = ("image",)
 
 
 class MovieListSerializer(MovieSerializer):
```

Comment:
what do you inherit from MovieSerializer?
There aren't any fields to inherit. If you want to inherit Meta class, then you must do this:

class Meta(MovieSerializer.Meta)